README.md

```diff
@@ -19,3 +20,33 @@ Currently supported packages:
 | **OpenSSH**      | [Github: oqs-demos/openssh](openssh)   | [Dockerhub: openquantumsafe/openssh](https://hub.docker.com/repository/docker/openquantumsafe/openssh)                       |
 
 You can use the curl and Chromium clients with the Open Quantum Safe test server at https://test.openquantumsafe.org/.
+
+## Contributing
+
+Contributions are gratefully welcomed. See our [Contributing Guide](https://github.com/open-quantum-safe/oqs-demos/wiki/Contributing-guide) for more details.
+
+## License
+
+All modifications to this repository are released under the same terms as [liboqs](https://github.com/open-quantum-safe/liboqs), namely as described in the file [LICENSE](https://github.com/open-quantum-safe/liboqs/blob/main/LICENSE.txt).
+
+## Team
+
+The Open Quantum Safe project is led by [Douglas Stebila](https://www.douglas.stebila.ca/research/) and [Michele Mosca](http://faculty.iqc.uwaterloo.ca/mmosca/)at the University of Waterloo.
+
+## Contributors to oqs-demos include:
+
+    Christian Paquin (Microsoft Research)
+    Dimitris Sikeridis (University of New Mexico / Cisco Systems)
+    Douglas Stebila (University of Waterloo)
+    Goutam Tamvada (University of Waterloo)
+    Michael Baentsch (baentsch.ch)
+    ISE @ FHNW (Fachhochschule Nordwestschweiz)
```

Comment:
Do you want to put some names or expand out what ISE stands for?

Reply:
This formatting was intentional: I got feedback that the latest additions were a team effort even though only one person was the github reference; that's why all individuals are listed in the openssh README. ISE @ FHNW written out is a pretty long German term ("Institut für Sensorik und Elektronik, Fachhochschule Nordwestschweiz") I felt breaks the usual length/level of "affiliation detail" at this place. Lastly Google gets you there immediately if interested. Please make a change proposal if you feel this should be different: My first goal is to get this through CI and deployed at Docker Hub and fine-tune later.